docs/ref/validators.txt

```diff
@@ -340,9 +340,14 @@ to, or in lieu of custom ``field.clean()`` methods.
 ``StepValueValidator``
 ----------------------
 
-.. class:: StepValueValidator(limit_value, message=None)
+.. class:: StepValueValidator(limit_value, offset=None, message=None)
 
     Raises a :exc:`~django.core.exceptions.ValidationError` with a code of
     ``'step_size'`` if ``value`` is not an integral multiple of
     ``limit_value``, which can be a float, integer or decimal value or a
-    callable.
+    callable. When ``offset`` is set, the validation occurs against
+    ``limit_value`` plus that offset.
```

Comment:
Just a suggestion but try reading this aloud to see if it's more comprehensible (gh won't let me suggest on deleted lines):

> Raises a ValidationError with a code of 'step_size' if value is not an integral multiple of limit_value + offset, if provided, which can be a float...

If putting 0 as the default in the signature then the sentence is clearer:

> ... with a code of 'step_size' if value is not an integral multiple of limit_value + offset, which can be ...